src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,15 @@
-package core.basesyntax;
-
-public class Application {
-    public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
-    }
-}
+package core.basesyntax;
+
+public class Application {
+    public static void main(String[] args) {
+        // create three balls using class Lottery and print information about them in console
+        Ball [] setOfBalls =
+                {new Lottery().getRandomBall(),
+                 new Lottery().getRandomBall(),
+                 new Lottery().getRandomBall()};
+
+        for (Ball ball : setOfBalls) {
+            System.out.println(ball.toString());

```

Comment:
```suggestion
            System.out.println(ball);
```
Method  `toString() ` will be called automatically